docs/en/integrations/data-ingestion/dbms/postgresql/data-type-mappings.md

```diff
@@ -35,7 +35,7 @@ The following table shows the equivalent ClickHouse data types for Postgres.
 | CIDR | [String](/en/sql-reference/data-types/string) |
 | HSTORE | [Map(K, V)](/en/sql-reference/data-types/map), [Map](/en/sql-reference/data-types/map)(K,[Variant](/en/sql-reference/data-types/variant)) |
 | UUID | [UUID](/en/sql-reference/data-types/uuid) |
-| ARRAY<T\> | [ARRAY(T)](/en/sql-reference/data-types/array) |
+| ARRAY&lt;T\> | [ARRAY(T)](/en/sql-reference/data-types/array) |
```

Comment:
nit: `\<` will look better if we can use it

Reply:
in some cases it rendered \ annoyingly, lets do a replace post upgrade and see if works then